src/js/bs/src/util/backdrop.js

```diff
@@ -56,10 +43,19 @@ class Backdrop {
     if (this._config.isAnimated) {
       reflow(this._getElement());
     }
-
-    this._getElement().classList.add(CLASS_NAME_SHOW);
-    this._element.setAttribute(`data-te-backdrop-${CLASS_NAME_SHOW}`, '');
-    this._config.isAnimated && this._element.animate(ANIMATION_FADE_IN, ANIMATION_TIMING);
+    this._getElement().classList.add('opacity-50');
+    this._getElement().classList.add('transition-all');
+    this._getElement().classList.add('duration-300');
+    this._getElement().classList.add('ease-in-out');
+    this._getElement().classList.add('fixed');
+    this._getElement().classList.add('top-0');
+    this._getElement().classList.add('left-0');
+    this._getElement().classList.add('z-[1050]');
+    this._getElement().classList.add('bg-[#000]');
+    this._getElement().classList.add('w-[100vw]');
+    this._getElement().classList.add('h-[100vh]');
```

Comment:
There should be a helper function to add multiple classes at once. If there's not, then we should add such a function.